### PR TITLE
Add viewport-units blogpost (#8864)

### LINKED ISSUE #8864: [CSS] Meet the new Viewport Units [11/29/2022]
* Proposed title: Meet the new Viewport Units
* Proposed by: bramus
* Related areas: CSS
* Content type: None of the above
* Deadline: 11/29/2022
* Deadline explanation: Chrome 108 Release Date
* [Draft](https://docs.google.com/document/d/1EX-lcUe0-mXPscqRXg5zI-5OTbMQYxAxplEwzEHqm9o/edit)
* Content summary: Post about the Large, Small, and Dynamic Viewport Units

### PR DESCRIPTION
**Pull request by bramus**
Fixes #8864.

Feel free to adjust the publication date, as long as it’s before/around the M108 release date _(2022-11-29)_.

### CODE REVIEW

**Comment by bramus** [comment]
The compat table used in this post relies on this PR for correctness: https://github.com/mdn/browser-compat-data/pull/18231

It’s not approved/merged at this moment.

**Comment by Th3S4mur41** [comment]
@bramus we had the same idee at almost the same time 😉
https://github.com/mdn/browser-compat-data/pull/18230

File: src/site/content/en/blog/viewport-units/index.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title: The Large, Small, and Dynamic Viewport units
+subhead: >
+  Meet the new viewport units which come in handy on mobile.
+authors:
+  - bramus
+description: >
+  Meet the new viewport units which come in handy on mobile.
```
Comment (rachelandrew) [inline src/site/content/en/blog/viewport-units/index.md:9]:
```suggestion
  New CSS units that account for mobile viewports with dynamic toolbars.
```

**Review by rachelandrew — CHANGES_REQUESTED** [review]
Just removing some emphasis as per styleguide and a suggestion for the description as "come in handy" isn't especially descriptive. Though I'm not set on what I've put if you have a better idea!

**Comment by bramus** [comment]
Good suggestions, thanks.

**Review by rachelandrew — APPROVED** [review]

File: src/site/content/en/blog/viewport-units/index.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title: The Large, Small, and Dynamic Viewport units
```
Comment (rachelandrew) [inline src/site/content/en/blog/viewport-units/index.md:3]:
```suggestion
title: The large, small, and dynamic viewport units
```

Comment (bramus) [reply src/site/content/en/blog/viewport-units/index.md]: I know, my capitalisation isn’t The Best™ 🙈

Will try and remember for next time 😊

Comment (rachelandrew) [reply src/site/content/en/blog/viewport-units/index.md]: I'd missed it earlier, I was just writing the web platform post which made me look at the title :)

**Pull request merged**